tests/integration/helpers.test.js

```diff
@@ -58,13 +58,14 @@ describe('GET /helpers', () => {
       { name: 'coach', expectedCode: 200 },
       { name: 'helper', expectedCode: 403 },
```

Comment:
le test should return 404 if invalid customer n'est pas utile il est compris dans le suivant

Reply:
done